backstop/tests/tree.html

```diff
@@ -0,0 +1,1611 @@
+                  incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                  ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+                  velit
+                  esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+                  culpa
+                  qui officia deserunt mollit anim id est laborum.
+                </div>
+                <div class="iui-caption"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                  incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                  ullamco
+                  laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate
+                  velit
+                  esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+                  culpa
+                  qui officia deserunt mollit anim id est laborum.
+                </div>
+              </span>
+            </div>
+          </div>
+          <ul class='iui-sub-tree'>
+            <li>
+              <div
+                class="iui-tree-node"
+                style='--level: 1;'
+                tabindex="0"
+                aria-expanded="false"
+              >
+                <div class="iui-tree-node-content">
+                  <button
+                    class="iui-button iui-borderless iui-small iui-expander"
+                    type="button"
+                  >
+                    <svg-chevron-right
+                      class="iui-button-icon"
+                      aria-hidden="true"
+                    ></svg-chevron-right>
+                  </button>
+                  <span class="iui-tree-node-content-label">
+                    <div class="iui-title"> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                      tempor
+                      incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                      ullamco
+                      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                      voluptate
+                      velit
+                      esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt
+                      in
+                      culpa
+                      qui officia deserunt mollit anim id est laborum.
+                    </div>
+                  </span>
+                </div>
+              </div>
+            </li>
+            <li>
+              <div
+                class="iui-tree-node"
+                style='--level: 1;'
+                tabindex="0"
+                aria-expanded="false"
+                id="test-node"
```

Comment:
Can you move this `id="test-node"` down 2 rows?  That way we're testing on a row that has the secondary line of text.
![Screen Shot 2021-12-01 at 4 23 44 PM](https://user-images.githubusercontent.com/849817/144316256-1d031d0e-2601-4ff7-8304-414e708c0dfb.png)

Reply:
For some reason test was not working on that node, but I moved test node to the first node that also has a caption